CHANGELOG.md

```diff
@@ -27,6 +27,10 @@ Currently, this repo is in Prerelease. When it is released, this project will ad
 - Fixes the sizing of the `errorFilled` and `errorOutline` icons in the `Icon` component.
 - Adds new props to Chakra's `ToolTip` to more forcefully close the DS `ToolTip`. New props include `closeDelay`, `closeOnClick`, `closeOnEsc`, and `closeOnMouseDown`.
 
+### Removes
+
+- Removes the `Button` component's warning about the `link` variant being deprecated.
```

Comment:
This is a temporary change, right? Maybe we should note that this warning will be added back at some point?

Reply:
Yes, we can say it's temporary, but I can't say when we might add it back.

Reply:
Let me know what you think of the latest.